pkg/csi/service/common/common_controller_helper.go

```diff
@@ -297,3 +297,23 @@ func IsvSphere8AndAbove(ctx context.Context, aboutInfo vim25types.AboutInfo) (bo
 	// For all other versions.
 	return false, nil
 }
+
+// CheckPVtoVVOLSupport internally checks if the vCenter version is 7.0.2
```

Comment:
Mention that this supports both vSAN and vVol.

I'm thinking we should rename "PV to VVol Mapping" to "PV to BackingDiskId Mapping ".  Otherwise it is confusing for vSAN users.